packages/search/package.json

```diff
@@ -24,7 +24,7 @@
     "ember-auto-import": "^1.5.3",
     "ember-cli-babel": "^7.17.2",
     "ember-cli-htmlbars": "^4.2.0",
-    "ember-cli-mirage": "^1.1.6",
+    "ember-cli-mirage": "~0.4.9",
```

Comment:
Why did mirage get downgraded?

Reply:
I had some issues with Pretender and I downgraded it to the version we use in the other packages.